kokoro/windows/protoc/release.cfg

```diff
@@ -0,0 +1,8 @@
+# Configuration for Windows protoc release builds
+build_file: "protobuf/kokoro/windows/protoc/build.bat"
```

Comment:
For the file path, how about a dedicated release directory? For example:
protobuf/kokoro/release/protoc/windows/...
protobuf/kokoro/release/python/...
protobuf/kokoro/release/java/...

Reply:
That sounds good to me. I just now rearranged it to match that directory structure.